main/src/cgeo/geocaching/log/LogEntry.java

```diff
@@ -80,9 +80,9 @@
         /** Friend's log entry */
         private boolean friend;
         /** report problem */
-        private ReportProblemType reportProblem;
+        @NonNull private ReportProblemType reportProblem;
```

Comment:
This looks unrelated to the commit message. Can you put it (and the setReportProblem change) into a separate commit?

Reply:
Sure